.coveragerc

```diff
@@ -67,6 +67,7 @@ omit =
     homeassistant/components/camera/xiaomi.py
     homeassistant/components/camera/yi.py
     homeassistant/components/cast/*
+    homeassistant/components/climate/coolmaster.py
```

Comment:
Eg that would be `homeassistant/components/coolmaster/climate.py`.

Reply:
I couldn't find any documentation for this. I did however see a script by @balloob that's supposed to do this in mass, and it seems like it hasn't run for climate yet since there are other platforms in there. Can we keep it like this and have the script change it when run?

Reply:
Yes, it's ok to keep for now. It's just a heads up. We probably will merge this before the climate platforms are converted.